CONTRIBUTING.md

```diff
@@ -1,146 +1,107 @@
-# Contributing to `telnyx-node`
+## Setting up the environment
 
-We'd love for you to contribute to our source code and to make `telnyx-node`
-even better than it is today! Here are the guidelines we'd like you to follow:
+This repository uses [`yarn@v1`](https://classic.yarnpkg.com/lang/en/docs/install).
```

Comment:
yarn 1 is legacy. It's better to use the latest yarn version

Reply:
so, migrating to latest yarn is a huge breaking change, and it'd be hard to do that across all of our typescript libraries

we have a config option to switch to pnpm if you want, https://www.stainless.com/docs/reference/config#type-script , it's targets.typescript.options.package_manager

Reply:
@lucaspcmr let's use pnpm instead